spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :user do
+    email { 'test@test.com' }
```

Comment:
you can use [faker](https://github.com/faker-ruby/faker) to generate random data :)

Reply:
and what would be the benefits of using it? I mean, since we would be adding a new Gem 😅

Reply:
You can avoid using that gem but in that case, you need to use the sequence method as it is mentioned [here](https://arjanvandergaag.nl/blog/factory_girl_tips.html). 

Your email is unique and if you need to create more than one user it will fail. 
Also, I think the uniqueness validation in the user model is missing 😅

Reply:
You're right about the sequence, since I had no need for it yet I didn't add it but I'll add it now :)

About the uniqueness validation, it is not missing, it's just not explicit in the user model since it was created with `Devise`, but there's a db constraint in the migration.

Reply:
oh you're right! cool 👍